WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -428,6 +449,48 @@ - (NSString *)titleForHeaderInSection:(NSInteger)section
     return headingTitle;
 }
 
+- (void)showPrivacySelector
+{
+    NSArray *values = @[ @(BlogPrivacyPublic), @(BlogPrivacyHidden), @(BlogPrivacyPrivate)];
+    NSMutableArray *titles = [NSMutableArray array];
+    for (NSNumber * value in values) {
+        [titles addObject:[self.blog textForBlogPrivacy:[value integerValue]]];
+    }
+    NSArray *hints = @[
+                       NSLocalizedString(@"Your site is visible to everyone, and it maybe indexed in search engines.", @"Hint for users when public privacy setting is set"),
```

Comment:
Small correction - `and it maybe` to `and it may be` also `indexed in` to `indexed by`